os/include/tinyara/meminfo_drv.h

```diff
@@ -16,22 +16,20 @@
  *
  ****************************************************************************/
 
-#ifndef __INCLUDE_TINYARA_HEAPINFO_DRV_H
-#define __INCLUDE_TINYARA_HEAPINFO_DRV_H
+#ifndef __INCLUDE_TINYARA_MEMINFO_DRV_H
+#define __INCLUDE_TINYARA_MEMINFO_DRV_H
 
 /****************************************************************************
  * Included Files
  ****************************************************************************/
 #include <tinyara/config.h>
-#ifdef CONFIG_APP_BINARY_SEPARATION
-#include <tinyara/binary_manager.h>
-#endif
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define HEAPINFO_DRVPATH     "/dev/heapinfo"
+#define MEMINFO_DRVPATH     "/dev/meminfo"
 
+#if defined(CONFIG_DEBUG_MM_HEAPINFO)
```

Comment:
I'm not sure heapinfo structure is in meminfo header.

Reply:
Yes. I agree with your opinion. I will consider more.